test/test_prims.py

```diff
@@ -1094,6 +1094,35 @@ def test_reshape_view_method(self, device, dtype):
         result_refs = refs.view(a, *new_shape)
         self.assertEqual(result_eager, result_refs)
 
+    @dtypes(torch.float32)
+    def test_as_strided_scatter(self, device, dtype):
+        make_arg = partial(make_tensor, device=device, dtype=dtype)
+
+        inp = make_arg((4, 1))
+        # as_strided_scatter in prim is only making a clone of input
+        # and doesn't copy from src, clone src of itself for testing
```

Comment:
wdym? as_strided_scatter in prim is correctly coying the src:
```
In [26]: a=torch.ones(4,4)

In [27]: src = torch.ones(2,3)*2

In [28]: torch._prims.as_strided_scatter(a, src, (2,3), (3,1), 0)
Out[28]: 
tensor([[2., 2., 2., 2.],
        [2., 2., 1., 1.],
        [1., 1., 1., 1.],
        [1., 1., 1., 1.]])

Reply:
You are right, I think I miss interpret the _as_strided_scatter_meta as inside it doesn't copy the src.

https://github.com/pytorch/pytorch/blob/14ccad73b48edde1d4b41d26668c324424a531d7/torch/_prims/__init__.py#L1832

I'll update the comment.